kernels/volk/volk_32fc_x2_s32fc_multiply_conjugate_32fc.h

```diff
@@ -0,0 +1,247 @@
+#endif /* LV_HAVE_SSE */
+
+#endif /* INCLUDED_volk_32fc_x2_s32fc_multiply_conjugate_32fc_a_H */
+
+
+#ifndef INCLUDED_volk_32fc_x2_s32fc_multiply_conjugate_32fc_u_H
+#define INCLUDED_volk_32fc_x2_s32fc_multiply_conjugate_32fc_u_H
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <volk/volk_complex.h>
+#include <float.h>
+
+
+#ifdef LV_HAVE_NEON
+#include  <arm_neon.h>
+
+static inline void volk_32fc_x2_s32fc_multiply_conjugate_32fc_neon(lv_32fc_t* cVector, const lv_32fc_t* aVector, const lv_32fc_t* bVector, const lv_32fc_t scalar, unsigned int num_points){
+    const lv_32fc_t* bPtr = bVector;
+    const lv_32fc_t* aPtr = aVector;
+    lv_32fc_t* cPtr = cVector;
+    unsigned int number = num_points;
+    unsigned int quarter_points = num_points / 4;
+
+    float32x4x2_t a_val, b_val, c_val, scalar_val;
+    float32x4x2_t tmp_val;
+
+    scalar_val.val[0] = vld1q_dup_f32((const float*)&scalar);
+    scalar_val.val[1] = vld1q_dup_f32(((const float*)&scalar) + 1);
+
+    for(number = 0; number < quarter_points; ++number) {
+        a_val = vld2q_f32((float*)aPtr);
+        b_val = vld2q_f32((float*)bPtr);
+        b_val.val[1] = vnegq_f32(b_val.val[1]);
+        __VOLK_PREFETCH(aPtr + 8);
+        __VOLK_PREFETCH(bPtr + 8);
+
+        tmp_val.val[1] = vmulq_f32(b_val.val[1], scalar_val.val[0]);
+        tmp_val.val[0] = vmulq_f32(b_val.val[0], scalar_val.val[0]);
+
+        tmp_val.val[1] = vmlaq_f32(tmp_val.val[1], b_val.val[0], scalar_val.val[1]);
+        tmp_val.val[0] = vmlsq_f32(tmp_val.val[0], b_val.val[1], scalar_val.val[1]);
+
+        c_val.val[1] = vaddq_f32(a_val.val[1], tmp_val.val[1]);
+        c_val.val[0] = vaddq_f32(a_val.val[0], tmp_val.val[0]);
+
+        vst2q_f32((float*)cPtr, c_val);
+
+        aPtr += 4;
+        bPtr += 4;
+        cPtr += 4;
+    }
+
+    for(number = quarter_points*4; number < num_points; number++){
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+    }
+}
+#endif /* LV_HAVE_NEON */
+
+#ifdef LV_HAVE_GENERIC
+
+static inline void volk_32fc_x2_s32fc_multiply_conjugate_32fc_generic(lv_32fc_t* cVector, const lv_32fc_t* aVector, const lv_32fc_t* bVector, const lv_32fc_t scalar, unsigned int num_points){
+    const lv_32fc_t* aPtr = aVector;
+    const lv_32fc_t* bPtr = bVector;
+    lv_32fc_t* cPtr = cVector;
+    unsigned int number = num_points;
+
+    // unwrap loop
+    while (number >= 8){
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
+        number -= 8;
+    }
+
+    // clean up any remaining
+    while (number-- > 0)
+        *cPtr++ = (*aPtr++) + lv_conj(*bPtr++) * scalar;
```

Comment:
I'm not sure but we mostly use `{}` for one-liners. It's something I prefer and recommend at least.

Reply:
I will use {}.